src/components/ListItem/styles.module.css

```diff
@@ -12,12 +11,16 @@
 }
 
 .listItem input {
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+    sans-serif;
   background-color: transparent;
   border: 0;
-  font-size: 12px;
-  font-weight: 700;
+  font-size: 16px;
+  font-weight: 600;
   flex-grow: 1;
   cursor: pointer;
+  color: var(--font_grey);
```

Comment:
I think this grey on grey text makes it quite faint

Reply:
I will make the font grey color darker so that there's a clearer distinction.